source/infrastructure/free-radius/healthcheck.html.md.erb

```diff
@@ -0,0 +1,30 @@
+---
+title: FreeRADIUS Healthcheck
+weight: 100
+last_reviewed_on: 2022-02-21
+review_in: 6 months
+---
+
+# The FreeRADIUS Healthcheck
+
+The [Route53 healthcheck](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-failover.html) connects to port `3000` on a specific FreeRADIUS servers. For example:
+```
+http://12.345.678.91:3000/
```

Comment:
Unfortunately, the changes I've been making have made this out of date, as the Route53 healthchecks are no more.

Instead, the frontend target groups health check is what's connecting to the tasks on port 3000.